include/admin/comments.inc.php

```diff
@@ -9,23 +9,23 @@
 $commentsPerPage = (int)(!empty($serendipity['GET']['filter']['perpage']) ? $serendipity['GET']['filter']['perpage'] : 10);
 $summaryLength = 200;
 
-$errormsg = '';
-$msg = '';
+array ($errormsg);
+array ($msg);
```

Comment:
I would write these lines as:

    $errormsg = array();
    $msg = array();

That way it is clear that we create a new and empty array here. array($var) reads like casting an existing variable to be an array.